js/classes/sub-set.js

```diff
@@ -0,0 +1,99 @@
+// Variable containing availablity and exchange specific pair identifier for
+//    all exchanges
+// TODO: Begin filling this variable out
+var info = {
+  poloniex : {
+    BTCUSD : "USDT_BTC",
+    ETHBTC : "BTC_ETH"
+  },
+  gdax : {
+    BTCUSD : "BTC-USD",
+    ETHBTC : "ETH-BTC"
+  },
+  kraken : {
+    BTCUSD : "XXBTZUSD",
+    ETHBTC : "XETHXXBT"
+  },
+  bitfinex : {
+    BTCUSD : "BTCUSD",
+    ETHBTC : "ETHBTC"
+  }
+}
+
+// Function which takes in a orderbook-exchange pair, and opens a feed of data
+//    to that orderbook-exchange pair
+let open = function open(orderbook, exchange, pair){
```

Comment:
Seems like the parameters should go big to small like `exchange, orderbook, pair`

However you don't need to pass in `pair` since it's part of each `orderbook`